examples/tfengine/gke_cluster.hcl

```diff
@@ -0,0 +1,175 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# {{$recipes := "../../templates/tfengine/recipes"}}
+
+data = {
+  parent_type      = "organization" # One of `organization` or `folder`.
+  parent_id        = "12345678"
+  billing_account  = "000-000-000"
+  state_bucket     = "example-terraform-state"
+  storage_location = "us-central1"
+  compute_region   = "us-central1"
+}
+
+template "networks" {
+  recipe_path = "{{$recipes}}/project.hcl"
+  output_path = "./gke_cluster/networks"
+  data = {
+    project = {
+      project_id         = "networks-example-project"
```

Comment:
You can name the project `example-networks` to be consistent with other examples.

Reply:
Done.